### ISSUE TITLE
Cache.frominde

### ISSUE THREAD

**Issue by MikiGrit** [grantjenks/python-diskcache#271]

**Comment by grantjenks** [comment]
Yes, that’s true. Probably should be a bug. I would expect Deque and Index to set the eviction policy to none automatically.

**Comment by grantjenks** [comment]
I think the best fix is actually to emit a warning if `cache.eviction_policy == 'none'` when calling `fromcache`. I don't otherwise think `fromcache` should change any cache settings itself.

**Comment by grantjenks** [comment]
Another bug fix is to set the eviction policy to 'none' inside `FanoutCache.index`.
